drizzle-orm/src/sql/expressions/conditions.ts

```diff
@@ -22,6 +22,26 @@ export function bindIfParam(value: unknown, column: AnyColumn | SQL.Aliased): SQ
 	return value as SQLChunk;
 }
 
+
+/**
+ * Test that two values are equal.
+ *
+ * Remember that the SQL standard dictates that
+ * two NULL values are not equal, so if you want to test
+ * whether a value is null, you may want to use
+ * `isNull` instead.
+ * 
+ * ## Examples
+ *
+ * ```ts
+ * // Select cars made by Ford
+ * db.select(cars)
+ *   .where(
+ *     eq(cars.make, 'Ford'))
```

Comment:
Please change the formatting through the code chunks to be either like this:
```suggestion
 *   .where(
 *     eq(cars.make, 'Ford'),
 *   )
```
or like this:
```suggestion
 *   .where(eq(cars.make, 'Ford'))
```

Reply:
Yep, done